frontend/src/app/oe/components/preview/preview.component.ts

```diff
@@ -50,4 +50,8 @@ export class PreviewComponent implements OnInit {
   pastEnergyDetailLink() {
     return this.relativeUrlPipe.transform('/hashstrikes/energy_detail/89778/89791');
   }
+
+  futureEnergyDetailLink() {
+    return this.relativeUrlPipe.transform('/hashstrikes/energy_detail/89778/120000');
```

Comment:
it seems, that "future" URL has hardcoded block height from signet network. If that is intended, then it is ok.

Reply:
Yes, for now it is intended like other URLs